addons/html_builder/static/src/builder/plugins/grid_layout/grid_layout_plugin.js

```diff
@@ -0,0 +1,68 @@
+import { Plugin } from "@html_editor/plugin";
+import { setElementToMaxZindex } from "@html_builder/builder/utils/grid_layout_utils";
+import { isMobileView } from "@html_builder/builder/utils/utils";
+
+const gridItemSelector = ".row.o_grid_mode > div.o_grid_item";
+
+export class GridLayoutPlugin extends Plugin {
+    static id = "gridLayout";
+    static dependencies = ["overlay", "history"];
+    resources = {
+        has_overlay_buttons: this.isGridItem.bind(this),
```

Comment:
you define has_overlay_buttons but you never use it ?

Reply:
Same, anticipation for the future, I will remove it for now :+1: